omnigibson/simulator.py

```diff
@@ -684,6 +684,9 @@ def play(self):
                 # Take a render step -- this is needed so that certain (unknown, maybe omni internal state?) is populated
                 # correctly.
                 self.render()
+                
+                # During rendering, the Fabric API is updated, so we can mark it as clean
+                PoseAPI.mark_valid()
```

Comment:
if rendering automatically validates PoseAPI, should we bake this directly into the render() call?

Reply:
@hang-yin lets do this by overriding render. I think there might be another place where we do this post-render validation too.